s3/pkg/conf/tidb.sql

```diff
@@ -123,6 +123,26 @@ CREATE TABLE IF NOT EXISTS `multiparts` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `objectparts`
+--
+
+DROP TABLE IF EXISTS `objectparts`;
```

Comment:
With this drop operation, every time restart the docker will remove those data already exist.

Reply:
thanks, will delete the line